en/docs/setup/deployment-patterns.md

```diff
@@ -0,0 +1,210 @@
+</tr>
+<tr class="odd">
+<td>Directory services</td>
+<td><p>Supports Directory Services implementing following LDAP Protocols:</p>
+<ul>
+<li>LDAP v2</li>
+<li>LDAP v3</li>
+</ul>
+<p>For information on tested LDAPs, see <a href="https://docs.wso2.com/display/compatibility/Tested+LDAPs">Tested LDAPs</a> .</p></td>
+</tr>
+<tr class="even">
+<td>Java</td>
+<td>Oracle JDK 1.8 (There’s a <a href="https://bugs.openjdk.java.net/browse/JDK-8189789">known issue</a> with JDK1.8.0_151)</td>
+</tr>
+<tr class="odd">
+<td>Web browsers</td>
+<td><p>For more information on tested web browsers, see <a href="https://docs.wso2.com/display/compatibility/Tested+Web+Browsers">Tested Web Browsers</a> .</p></td>
+</tr>
+<tr class="even">
+<td>Load balancers</td>
+<td><p>For more information about load balancers, see <a href="https://docs.wso2.com/display/CLUSTER44x/Setting+up+a+Cluster#SettingupaCluster-Configuringtheloadbalancer">Configuring the load balancer</a> .</p></td>
+</tr>
+</tbody>
+</table>
+
+### Deployment Patterns
+
+-   [Pattern 1 - HA clustered deployment of WSO2 Identity
+    Server](#DeploymentPatterns-Pattern1-HAclustereddeploymentofWSO2IdentityServer)
+-   [Pattern 2 - HA clustered deployment of WSO2 Identity Server with
+    WSO2 Identity
+    Analytics](#DeploymentPatterns-Pattern2-HAclustereddeploymentofWSO2IdentityServerwithWSO2IdentityAnalytics)
+
+!!! note
+    
+    Notes
+    
+    Note the following before you begin:
+    
+    1.  Ensure high availability for the respective RDMS and Directory
+        Services used for each of the deployment patterns given below.
+    
+    2.  In each production deployment, share the runtime deployment
+        artifacts among nodes using a shared file system. In the deployment
+        patterns defined below, this process is referred to as ' Artifact
+        synchronization ’.
+    
+        The Runtime deployment artifacts are:
+    
+        1.  Email output event publisher  
+            /repository/deployment/server/eventpublishers/
+    
+        2.  Secondary user stores  
+            /repository/deployment/server/userstores/
+    
+        3.  Analytics data publishers and event streams  
+            /repository/deployment/server/eventpublishers/  
+            /repository/deployment/server/eventstreams/
+    
+        4.  Workflow engine related artifacts  
+            /repository/deployment/server/humantasks/  
+            /repository/deployment/server/bpel/
+    
+
+#### Pattern 1 - **HA clustered deployment of WSO2 Identity Server**
+
+This deployment can be scaled from two to N nodes based on capacity
+requirements.
+
+-   Load balancer should be configured to use sticky sessions
+
+-   All WSO2 Identity Server nodes should participate in a cluster.
+    Clustering is used to invalidate local caches of nodes, by notifying
+    over cluster messages, as cache updates happen.
+
+    ![](attachments/103329471/103329475.png)   
```

Comment:
Please rename the image and folder path for this. Do this for all instances.

Reply:
The respective file was removed from the commit as the file is subject to change—it will be reflected in a subsequent PR.